### DISCUSSION TITLE
No option to invite friend?

### DISCUSSION THREAD

**Discussion by pitmetak** [m4dEngi/RemotePlayWhatever#23] [Q&A]
I'm trying to invite my friend to Risk of Rain 2 since I have a mod that creates splitscreen. Issue is, even after running the RemotePlayWhatever.exe, there's no option to invite them to the game? I right click my friend, and there's only "invite to watch". Help?

**Comment by m4dEngi** [comment]
RPW has its own menu for that in your systems tray. It does not change/patch steam client to extend its functionality.
